data-science-onramp/data-cleaning/requirements-test.txt

```diff
@@ -0,0 +1,4 @@
+pytest==6.0.0
+pandas==1.1.0
+google-cloud-bigquery==1.26.1
+pyarrow==1.0.0
```

Comment:
What is `pyarrow` being used for?

Reply:
It is required to load a table from BigQuery. I get the following errors when I run without it:
```
ImportError: Unable to find a usable engine; tried using: 'pyarrow', 'fastparquet'.
A suitable version of pyarrow or fastparquet is required for parquet support.
```
```
PyarrowMissingWarning: Loading dataframe data without pyarrow installed is deprecated and will become unsupported in the future. Please install the pyarrow package.
```

Reply:
Sounds good, lgtm